common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/ElasticSearchConfig.scala

```diff
@@ -1,3 +1,3 @@
 package com.gu.mediaservice.lib.elasticsearch
 
-case class ElasticSearchConfig(alias: String, url: String, cluster: String, shards: Int, replicas: Int)
+case class ElasticSearchConfig(alias: String, accretorAlias: String, url: String, cluster: String, shards: Int, replicas: Int)
```

Comment:
had to look up what https://en.wiktionary.org/wiki/accretor meant - and even then not sure exactly what it means in this context - perhaps we could pick a more mainstream word to describe this alias string?

Reply:
I nearly also used the word [roche](https://en.wikipedia.org/wiki/Interacting_binary_star) as well, which may have given away that I didn't intend to actually go through with this confusing terminology.